lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -139,6 +143,17 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
     _drawBar(canvas, viewSize, barPath, barData);
   }
 
+  void drawBetweenBarsArea(Canvas canvas, Size viewSize, LineChartBarData fromBarData, LineChartBarData toBarData, BetweenBarsData betweenBarsData) {
```

Comment:
Also here, please retrieve the `fromBarData` and `toBarData` inside the function using `betweenBarsData`.